replay-protected-sighash.md

```diff
@@ -89,7 +95,7 @@ The sighash type is altered to include a 24-bit *fork id* in its most significan
   ss << ((GetForkID() << 8) | nHashType);
 ````
 
-This ensure that the proposed digest algorithm will generate different results on forks using different *fork ids*.
+This ensure that the proposed digest algorithm we use when SIGHASH_FORKID is set will generate different results on forks using different *fork ids*.
```

Comment:
Is "all over the place" maybe a bit extreme description of adding it in one place? Repeating it once is not diluting the message, but I have no issues with you rejecting this part.